couchbase-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -12,19 +12,19 @@
       "value": "Small"
     },
```

Comment:
@singhkay, I have just added the placeholders to the parameter file. Let's see if the test passes.